src/components/PathwaysList/PathwaysList.tsx

```diff
@@ -23,19 +27,46 @@ const PathwaysList: FC = () => {
     setOpen(false);
   }, []);
 
+  const selectFile = useCallback(
+    (files: FileList | undefined | null) => {
+      if (files?.length) addPathwayFromFile((files[0] as unknown) as File);
+    },
+    [addPathwayFromFile]
+  );
+
   return (
     <div className={styles.root}>
-      <Button
-        className={styles.createPathwayButton}
-        variant="contained"
-        color="primary"
-        startIcon={<FontAwesomeIcon icon={faPlus} />}
-        onClick={openNewPathwayModal}
-      >
-        Create Pathway
-      </Button>
+      <div className={styles.buttonRow}>
+        <Button
+          className={styles.createPathwayButton}
+          variant="contained"
+          color="primary"
+          startIcon={<FontAwesomeIcon icon={faFileImport} />}
+          onClick={(): void => setImportPathwayOpen(true)}
```

Comment:
Use a callback here instead of passing in an anonymous function (https://mitrecorp.slack.com/archives/CLH34KL6A/p1594649905003300)